source/utils/VectorInt64.h

```diff
@@ -22,37 +22,37 @@
 #include <iostream>
 #include <OgreVector3.h>
 
-const int mPrecisionDigits = 20;
-const int Unit = (1 << mPrecisionDigits);
+const int PRECISION_DIGITS = 20;
+const int UNIT = (1 << PRECISION_DIGITS);
```

Comment:
I've not looked close enough to know what UNIT is used for but when I said "Unit" was not a good name, "UNIT" is not much better ;-)

Something like VECTOR3INT64_UNIT seems better. The idea is to not have such common variable names to avoid possible conflicts (and usually, such conflicts are pretty hard to debug if you includ a header that only defines UNIT if it is not).